src/components/core/Banner/Banner.tsx

```diff
@@ -29,10 +31,13 @@ const Banner = ({ children, closable, ...rest }: Props) => {
       </panda.div>
       {closable && (
         <Button
-          onClick={() => setIsOpen(false)}
+          size="xs"
           p={1}
-          opacity={{ _hover: 0.8 }}
+          display="flex"
```

Comment:
Is this needed for any specific reason? Just curious!

Reply:
Sometimes the Icon has a tough time positioning center, so I have just been using this little hack when using the Icon component. To be honest, I didn't even try it without, just toss it in by default now.

Reply:
Hmmm ok. Default display is `inline-flex` for the `Button` component that is why I asked, would assume that would work just fine. Will have to investigate further on icons being centered! Will approve for now anyhow as this is a super nit detail and can just be handled in a different PR if we choose to lol.